.env.example

```
@@ -85,3 +85,5 @@ PAYPAL_APP_CLIENTID=
 PAYPAL_APP_SECRET=
 PAYPAL_WEBHOOK_ENDPOINT=asasasdasdfafrgdfsg
 PAYPAL_WEBHOOK_ID=
+# For contributor retrieval
+GITHUB_API_KEY=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The GITHUB_API_KEY key should go before the HF_TOKEN key

(UnorderedKey)